### ISSUE TITLE
Error when dealing with fields with numbers in them

### ISSUE THREAD

**Issue by Alxandr** [yaahc/displaydoc#13]
The following fails:

```rust
/// {foo1} {foo2}
#[derive(displaydoc::Display)]
pub struct Test {
    foo1: String,
    foo2: String,
}
```

Looking at the output from cargo expand, it would seem the numbers gets stripped from the field names:

```rust
    impl core::fmt::Display for Test {
        fn fmt(&self, formatter: &mut core::fmt::Formatter) -> core::fmt::Result 
    {
            #[allow(unused_variables)]
            let Self { foo1, foo2 } = self;
            formatter.write_fmt(::core::fmt::Arguments::new_v1_formatted(
                &["", " "],
                &match (&foo, &foo) {
                    (arg0, arg1) => [::core::fmt::ArgumentV1::new(
                        arg1,
                        ::core::fmt::Display::fmt,
                    )],
                },
                &[
                    ::core::fmt::rt::v1::Argument {
                        position: 0usize,
                        format: ::core::fmt::rt::v1::FormatSpec {
                            fill: ' ',
                            align: ::core::fmt::rt::v1::Alignment::Unknown,
                            flags: 0u32,
                            precision: ::core::fmt::rt::v1::Count::Implied,
                            width: ::core::fmt::rt::v1::Count::Implied,
                        },
                    },
                    ::core::fmt::rt::v1::Argument {
                        position: 0usize,
                        format: ::core::fmt::rt::v1::FormatSpec {
                            fill: ' ',
                            align: ::core::fmt::rt::v1::Alignment::Unknown,
                            flags: 0u32,
                            precision: ::core::fmt::rt::v1::Count::Implied,
                            width: ::core::fmt::rt::v1::Count::Implied,
                        },
                    },
                ],
            ))
        }
    }
```

Note the `&match (&foo, &foo)`.

**Comment by yaahc** [comment]
Thank you for pointing this one out! It should be fixed now with version `0.1.6`.

**Comment by Alxandr** [comment]
Great. For now I've just been renaming things as "first" and "second" instead :P.
